testHelpers/utils.ts

```diff
@@ -79,14 +86,18 @@ class Utils {
   }
 
   setContracts(
-    erc1155721: ERC1155ERC721,
+    //erc1155721: ERC1155ERC721,
```

Comment:
Please remove this line.

Reply:
Removed. Will be included in next commit.